rules/unauthenticated-service/raw.rego

```diff
@@ -0,0 +1,50 @@
+package armo_builtins
+
+import future.keywords.contains
+import future.keywords.if
+
+deny contains msga if {
+	service := input[_]
+	service.kind == "Service"
+
+	wl := input[_]
+	spec_template_spec_patterns := {"Deployment", "ReplicaSet", "DaemonSet", "StatefulSet", "Pod", "Job", "CronJob"}
+	spec_template_spec_patterns[wl.kind]
+	wl_connected_to_service(wl, service)
+
+	service_scan_result := input[_]
+	service_scan_result.kind == "ServiceScanResult"
+	service_name := service.metadata.name
+	has_unauthenticated_service(service_name, service.metadata.namespace, service_scan_result)
+
+	# Path to the workload spec.
+	path := "spec"
```

Comment:
Is this is the failing path? The whole spec section?

Reply:
What should be the path in this case?

Reply:
Maybe the service_scan_result should hold the fixed path/command